chandra_aca/tests/test_maude_decom.py

```diff
@@ -272,6 +273,38 @@ def test_partial_images():
         assert np.all(table[i]["IMG"].mask == mask[table[i]["IMGTYPE"]])
 
 
+def test_aca_images_chunks_1():
+    """Test that a fetch longer than the maude 3 hour limit works"""
+    start = "2023:001:00:00:01.000"  # times picked close to 4.1 image boundary
+    stop = "2023:001:03:30:01.000"  # times picked close to 4.1 image boundary
+    imgs = maude_decom.get_aca_images(start, stop)
```

Comment:
This does a 3.5 hour fetch, which is slow - but I couldn't figure out a way around that.

Reply:
In general, I see that as a helpful testing technique, but I'm not sure how it helps in this case.  I think I'd need to set a debug mode in get_aca_images or the like to actually see it was _using_ a different step_max.  And the whole point of the step max here is to do multiple fetches to get around https://github.com/sot/chandra_aca/blob/master/chandra_aca/maude_decom.py#L1146 , so we'd need to set _that_ by a separate var for testing (to show that you can fetch more than the get_aca_packets limit of data with a step size that is at most equal to the get_aca_packets limit).

Reply:
You could put `maude_fetch_times` into the returned `Table.meta` and then check that.

Reply:
Right - something like that could work as debug data that wouldn't be too ugly.